hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -836,7 +835,7 @@ public OmMultipartUploadListParts listParts(String volumeName,
           //if there are no parts, use the replicationType from the open key.
           if (isBucketFSOptimized(volumeName, bucketName)) {
             multipartKey =
-                getMultipartOpenKeyFSO(volumeName, bucketName, keyName,
+                metadataManager.getMultipartKeyFSO(volumeName, bucketName, keyName,
```

Comment:
I believe we should use the OMMultipartUploadUtils.getMultipartOpenKey() function here, since we want to look into the open key table and not the main table.

Reply:
Fixed.